rules/Php74/Rector/LNumber/AddLiteralSeparatorToNumberRector.php

```diff
@@ -87,7 +83,7 @@ public function run()
 CODE_SAMPLE
                     ,
                     [
-                        AddLiteralSeparatorToNumberRector::LIMIT_VALUE => 1_000_000,
+                        self::LIMIT_VALUE => 1_000_000,
```

Comment:
configurable example should use original class reference instead of `self::` to ease copy paste config

Reply:
https://github.com/rectorphp/rector-src/pull/6037